.github/CODEOWNERS

```diff
@@ -29,3 +48,84 @@ cmake/**/*Windows*    @compnerd
 lib/Basic/Windows     @compnerd
 stdlib/public/Windows @compnerd
 utils/*windows*       @compnerd
+
+# AST
+/include/swift/AST/               @hborla @slavapestov @xedin
+/include/swift/AST/*Conformance*  @slavapestov
+/include/swift/AST/*Distributed*  @ktoso
+/include/swift/AST/Evaluator*     @CodaFi @slavapestov
```

Comment:
@CodaFi Do you want to be mentioned to any of these?

Reply:
Works for me